openshift/ci-operator/generate-ci-config.sh

```diff
@@ -23,7 +23,7 @@ binary_build_commands: make install
 test_binary_build_commands: make test-install
 tests:
 - as: e2e
-  commands: "make test-e2e"
+  commands: "INTERNAL_REGISTRY=image-registry.openshift-image-registry.svc:5000 ENABLE_ADMISSION_WEBHOOKS=false make test-e2e"
```

Comment:
Changes in this file look like a step back. We don't need these variables on OpenShift 4